### PR TITLE
Improved Click-box on Asset Buttons

### PR DESCRIPTION
**Pull request by dills122**
## Description:

Improves the usability of the buttons within a users wallet's asset view. Initially the button's click box was only the actual icon, which was a rather small click-box.

## Fix

Created new shared component for Router Icon Links that expands the click-box to the entire component/box.

## Demo

Before:

https://user-images.githubusercontent.com/15662762/201785386-8129207a-ccda-4871-b190-6030c2308947.mov

After:

https://user-images.githubusercontent.com/15662762/201785402-9539c797-d0bc-4e27-9a17-844f3a06f789.mov

Fix for Issue: https://github.com/tallycash/extension/issues/2567

Latest build: [extension-builds-2622](https://github.com/tallycash/extension/suites/9325674654/artifacts/438231861) (as of Tue, 15 Nov 2022 22:59:51 GMT).

### CODE REVIEW

**Comment by 0xDaedalus** [comment]
Big fan of this change - definitely a UX win.  Want to cc our designer @VladUXUI to double check that the behavior here and the colors used on hover is in line with his expectations.

File: ui/components/Shared/SharedIconRouterLink.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        .router_link_container {
+          margin: auto 4px;
+          border-radius: 4px;
+        }
+        .icon_wrapper {
+          display: flex;
+          padding: 0.5em;
+        }
+        .router_link_container:hover {
+          background-color: var(--white);
```
Comment (VladUXUI) [inline ui/components/Shared/SharedIconRouterLink.tsx:34]: Let's use var(--hunter-green) instead of the white.

**Review by VladUXUI — DISMISSED** [review]

**Comment by VladUXUI** [comment]
Overall i love this! the mouse over is not what we generally use, but with the mentioned change, i'm happy to push!

**Comment by dills122** [comment]
Screenshot with updated colors:
Screen Shot 2022-11-15 at 11 35 22 AM

**Review by VladUXUI — COMMENTED** [review]
Looks good on my end! Will let an engineer merge!

**Review by 0xDaedalus — DISMISSED** [review]
Tested - looks good, feels good, let's rock! 🚀 

Thanks for the contribution, and don't forget to claim your [GitPOAP](https://www.gitpoap.io/gp/579) 😄

**Comment by 0xDaedalus** [comment]
Hey @dills122 this PR is ready to go but it looks like commit https://github.com/tallycash/extension/pull/2622/commits/f68d71eb0f66a9a88448dc2d520c2cb2cbd9dfe7 is unsigned.

We do require commits to be signed prior to including them in main - so if you wouldn't mind editing the git history here to get that commit signed that would be the last thing needed to get this PR merged.

Since it looks like you already have signing set up per https://github.com/tallycash/extension/pull/2622/commits/b9695e764468f568b0c62d932eb7f914046f1c83 the easiest path here might be to `git rebase HEAD~3 -i` and edit the commit you want to sign via `e` in the interactive rebase prompt.  Alternatively feel free to rebase and squash the 2 commits into 1 - which should also trigger a new commit and a new signature.

Let me know if there's anything I can do to help here - getting signing to work on a past commit  can sometimes be a bit tricky.

**Comment by dills122** [comment]
> Hey @dills122 this PR is ready to go but it looks like commit [f68d71e](https://github.com/tallycash/extension/commit/f68d71eb0f66a9a88448dc2d520c2cb2cbd9dfe7) is unsigned.
> 
> We do require commits to be signed prior to including them in main - so if you wouldn't mind editing the git history here to get that commit signed that would be the last thing needed to get this PR merged.
> 
> Since it looks like you already have signing set up per [b9695e7](https://github.com/tallycash/extension/commit/b9695e764468f568b0c62d932eb7f914046f1c83) the easiest path here might be to `git rebase HEAD~3 -i` and edit the commit you want to sign via `e` in the interactive rebase prompt. Alternatively feel free to rebase and squash the 2 commits into 1 - which should also trigger a new commit and a new signature.
> 
> Let me know if there's anything I can do to help here - getting signing to work on a past commit can sometimes be a bit tricky.

Hey @0xDaedalus thanks for pointing that out, looks like I didn't have my git setup to always sign on my laptop.

**Pull request merged**